ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-main.yaml

```diff
@@ -273,6 +273,45 @@ tests:
   container:
     from: operator-sdk-mto
   skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Za-z]+\.md$)|((^|/)\.github/)|((^|/)\.tekton/)
+- always_run: false
+  as: gcp-ovn-mto-heterogeneous-perfscale
+  cluster: build10
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-initial
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: n2-standard-4
+      ADDITIONAL_WORKERS: "12"
+      COMPUTE_NODE_REPLICAS: "12"
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      MTO_OPERATOR_INSTALL_METHOD: catalog
+      OCP_ARCH: arm64
+    test:
+    - ref: ipi-install-heterogeneous
+    - ref: multiarch-tuning-operator-deploy
+    - ref: multiarch-tuning-operator-deploy-pod-placement
+    - ref: openshift-qe-node-density
+    workflow: ipi-gcp-ovn
```

Comment:
Is there a way to setup the number of pods/s we create (or similar)?

Reply:
Yes, it can be set by env [PODS_PER_NODE](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/node-density/openshift-qe-node-density-ref.yaml#L32) , it is 245 by default if you don't set.

Reply:
Added env PODS_PER_NODE to the job as 245, it can be overwritten if needed.